frontend/src/pages/course/Course.tsx

```diff
@@ -22,7 +25,31 @@ const Course: FC = () => {
   const isCourseAdmin = useIsCourseAdmin()
   const navigate = useNavigate()
   const location = useLocation();
+  const [confirmLeaveVisible, setConfirmLeaveVisible] = useState<boolean>(false);
 
+  const handleLeaveConfirm = async () => {
+    try {
+      const result = await leaveCourse(course.courseId.toString());
+      if (result.success) {
+        message.success(result.message);
+        setTimeout(() => {
+          window.location.href = AppRoutes.HOME; // Redirect to home page after leaving course
+        }, 2000); // 2 seconds
+      } else {
+        message.error(result.message);
```

Comment:
Je gebruikt hier `message` geïmporteerd van ant design zelf. Het probleem is dat de theme niet werkt op deze message. Ik heb hiervoor mijn eigen "message provider" component gemaakt. Deze kan je als volgt krijgen:
```tsx
const MyComponent = () => {
// ...
  const { message } = useAppApi()
 // ...

}
```